ipatests/test_install/test_changeconf.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+
+from __future__ import absolute_import
+
+import pytest
+from ipaclient.install.ipachangeconf import IPAChangeConf
+
+
+@pytest.fixture(scope='session')
+def config_file(tmpdir_factory):
+    file = tmpdir_factory.mktemp('data').join('config_file.conf')
```

Comment:
``file`` is the name of a Python 2 builtin. In other places, we use ``filename`` to refer to a file name.

Reply:
My bad..